src/tasks/task3.js

```diff
@@ -0,0 +1,14 @@
+module.exports = arrayClothes => {
+  const result = arrayClothes.map(clothes => {
+    const { price, priceForPair, quantity } = clothes;
+
+    return {
+      ...clothes,
+      quantity: quantity || 0,
+      price: price || clothes.priceForPair,
+      priceForPair: priceForPair || `$${clothes.price.split('$')[1] * 2}`,
```

Comment:
So when there is no price for a pair it takes the double price, but when there is no price it takes a price for a pair.
If there is an *2 on one side then why there is no /2 on the other side?
I'm just curious why this logic is applied only for one case and skipped for another case.

Reply:
I think that socks not sell for one)

Reply:
Is I have to change?

Reply:
Yep, but the `price` is still there, right? Then something is wrong here.
This trick is more a logic riddle than a programming task.

Reply:
> Is I have to change it?
You may think of a better solution just in this case.
As you said before, there is no situation when you want to buy a sock or a single glov. So why do we have a `pariceForPair` at all?